src/flick/friend/serializers.py

```diff
@@ -28,3 +28,42 @@ class Meta:
         model = User
         fields = ("id", User.USERNAME_FIELD, "first_name", "last_name", "profile")
         read_only_fields = fields
+
+
+class FriendRequestSerializer(serializers.ModelSerializer):
+    """
+    Friend Request Serializer
+    """
+
+    to_user = FriendUserSerializer(read_only=True)
+
+    class Meta:
+        model = FriendshipRequest
+        fields = ("to_user", "created", "rejected", "viewed")
+        read_only_fields = fields
+
+
+class IncomingRequestSerializer(serializers.ModelSerializer):
+    """
+    Friend Request Serializer
+    """
+
+    from_user = FriendUserSerializer(read_only=True)
+
+    class Meta:
+        model = FriendshipRequest
+        fields = ("from_user", "created", "rejected", "viewed")
+        read_only_fields = fields
+
+
+class FriendshipSerializer(serializers.ModelSerializer):
+    """
+    Friend Serializer
+    """
+
+    from_user = FriendUserSerializer(read_only=True)
+
+    class Meta:
+        model = Friend
+        fields = ("from_user", "created")
```

Comment:
take a look at my other models where I have "created_at" and "updated_at" -- let's keep these fields consistent (my other models like Item and Show I think already have these fields built in on how they work)

Reply:
same as vivi's reply before--we are using the model `Friend` from `django-friendship`, and they have named their fields as `from_user`,`to_user` and `created`, such that we have to use the same names as fields. `from_user` means the user that sent the friend request, and `to_user` is the user that received the request